GenerateToolingFeed/Helper.cs

```diff
@@ -14,31 +14,46 @@ public class Helper
     {
         public static System.Net.Http.HttpClient HttpClient => _lazyClient.Value;
 
-        public static string GetDownloadLink(string os, string architecture, string cliVersion, bool isMinified = false)
+        public static string newReleaseVersion = string.Empty;
+
+        public static string GetDownloadLink(string os, string architecture, string cliVersion, bool isMinified = false, bool isNet8 = false)

```

Comment:
nit: worried about hard coding .NET versions here.

Reply:
have moved this to configuration-based feed generation.